wwwroot/inc/deviceconfig.php

```diff
@@ -4,6 +4,11 @@
 # framework. See accompanying file "COPYING" for the full copyright and
 # licensing information.
 
+// functions for HP Procurve switches (N.11 OS)
+require_once 'breeds/hpprocurveN1178.php';
+// functions for Cisco IOS 15 switches
+require_once 'breeds/ios15.php';
```

Comment:
`make install-applib` will not handle the new files if they are in a sub-directory.

Reply:
Move them one level up to avoid a new directory please.